spring-web/src/main/java/org/springframework/http/ResponseEntity.java

```diff
@@ -44,7 +44,7 @@
  * HttpStatus statusCode = entity.getStatusCode();
  * </pre>
  *
- * <p>Can also be used in Spring MVC, as the return value from a @Controller method:
+ * <p>This can also be used in Spring MVC as the return value from a @Controller method:
```

Comment:
```suggestion
 * <p>This can also be used in Spring MVC as the return value from a {@code Controller} method:
```
Not sure if this is also applicable here.